Assets/MRTK/Examples/Demos/EyeTracking/General/Profiles/EyeTrackingDemoConfigurationProfile.asset

```diff
@@ -13,7 +13,8 @@ MonoBehaviour:
   m_Name: EyeTrackingDemoConfigurationProfile
   m_EditorClassIdentifier: 
   isCustomProfile: 1
-  targetExperienceScale: 4
+  experienceSettingsProfile: {fileID: 11400000, guid: 1ccbb3b1d759d6a49b4161a958329d9d,
```

Comment:
Leaving this as 3 because there's previously no functional difference between Room and World scales, and I don't want to pollute the repository with more profiles.

Reply:
> there's previously no functional difference between Room and World scales

I don't believe that's true. World scale aligns with HoloLens, where the origin is at your head, while Room scale is able to put the origin on the floor. They're handled differently:

https://github.com/microsoft/MixedRealityToolkit-Unity/blob/ca6f802b2107f3275feb34ddf083bd23d9a65b49/Assets/MRTK/Services/BoundarySystem/XR2018/MixedRealityBoundarySystem.cs#L70-L79